cmd/worker/funcs_kmod.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-sigs/kernel-module-management/internal/worker"
+	"github.com/spf13/cobra"
+)
+
+func kmodLoadFunc(cmd *cobra.Command, args []string) error {
```

Comment:
what about moving the whole  kmodCmds from main to here? For me it would more sense to create a separate file for all cobra commands, instead just ffor one function used as a RunE function in one specific command

Reply:
Absolutely, but in a different PR. I kept the changes minimal in this one.